concent_api/core/views.py

```diff
@@ -69,6 +69,7 @@ def receive(request, _message):
         if last_delivered_message_status is None:
             return None
 
+        decoded_message_data = deserialize_message(last_delivered_message_status.message.data.tobytes())
```

Comment:
Please move to line 78 - it is not used before that line.

Reply:
:+1: